Common/FeatureFlags.swift

```diff
@@ -114,10 +116,24 @@ struct FeatureFlagConfiguration: Decodable {
         #endif
 
         // Swift compiler config is inverse, since the default state is enabled.
-        #if WALSH_INSULIN_MODEL_DISABLED
-        self.walshInsulinModelEnabled = false
+        #if SIRI_DISABLED
```

Comment:
Shouldn't the default state be `disabled` for this?

Reply:
Why?

Reply:
Siri authorization is currently only needed for activating presets via voice. I had this disabled because Tidepool Loop is not supporting DIY style override presets yet, but now that I think about it, perhaps we can update this to activate the old Pre-Meal and Activity presets. I'll create a Jira ticket for that.